Mathlib/RingTheory/MvPowerSeries/LinearTopology.lean

```diff
@@ -0,0 +1,180 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir, María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir, María Inés de Frutos-Fernández
+-/
+import Mathlib.Data.Finsupp.Interval
+import Mathlib.RingTheory.MvPowerSeries.PiTopology
+import Mathlib.Topology.Algebra.LinearTopology
+import Mathlib.RingTheory.TwoSidedIdeal.Operations
+
+/-! # Linear topology on the ring of multivariate power series
+
+- `MvPowerSeries.LinearTopology.basis`: the ideals of the ring of multivariate power series
+all coefficients the exponent of which is smaller than some bound vanish.
+
+- `MvPowerSeries.LinearTopology.basis_mem_nhds_zero` :
+  the two-sided ideals from `MvPowerSeries.LinearTopology.basis` are neighborhoods of `0`.
```

Comment:
Unless I'm missing something, this is no longer here. You could either remove this mention from the docstring, or add it back as a one-line consequence of the basis statement (using [Filter.HasBasis.mem_of_mem](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Filter/Bases.html#Filter.HasBasis.mem_of_mem)).

Reply:
I replaced that docstring with the one mentioning hasBasis_nhds_zero.